oclif.manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "5.14.12",
+  "version": "5.14.13",
```

Comment:
This was updated when I ran the build command. Not sure what `oclif` is used for tbh, but when I ran the tests after making my change, one of them failed because of this version, so I included this in the PR.